lib/import/brca/providers/oxford/oxford_handler.rb

```diff
@@ -180,19 +189,19 @@ def targeted_scope_from_nullscope(genotype, record)
             end
           end
 
-          def extract_exemptions_from_record(genotype, record)
+          def extract_exemptions_from_record(genotype, record, variantpathclass)
             return if record.mapped_fields['codingdnasequencechange'].nil?
 
             exemptions = record.mapped_fields['codingdnasequencechange']
-            if exemptions.scan(/c\./).size.positive?
+            if exemptions.scan('c.').size.positive?
               genotype.add_gene_location(exemptions.gsub(/[\[\]+=]+/, ''))
-              genotype.add_status(2)
+              non_pathogenic_variant?(genotype, variantpathclass)
             elsif exemptions.scan(/(?<delinsdup>del|ins|dup)/i).size.positive?
               genotype.add_variant_type($LAST_MATCH_INFO[:delinsdup])
               if exemptions.scan(/(?<exno>[0-9]+-[0-9]+)/i).size.positive?
                 genotype.add_exon_location($LAST_MATCH_INFO[:exno])
               end
-              genotype.add_status(2)
+              non_pathogenic_variant?(genotype, variantpathclass)
```

Comment:
method call for non_pathogenic_variant?(genotype, variantpathclass) can be taken out of if-elsif if in both cases its being called.

Reply:
This has been updated - see commit f1aa39b